data-reporting-service/organization-service/src/main/java/gov/cdc/etldatapipeline/organization/model/dto/org/OrganizationReporting.java

```diff
@@ -0,0 +1,76 @@
+package gov.cdc.etldatapipeline.organization.model.dto.org;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.PropertyNamingStrategies;
+import com.fasterxml.jackson.databind.annotation.JsonNaming;
+import gov.cdc.etldatapipeline.organization.model.DataRequiredFields;
+import lombok.Builder;
+import lombok.Data;
+
+import java.util.Set;
+
+@Data
+@Builder
+@JsonIgnoreProperties(ignoreUnknown = true)
+@JsonNaming(PropertyNamingStrategies.SnakeCaseStrategy.class)
+public class OrganizationReporting implements DataRequiredFields {
+    private Long organizationUid;
+    private String localId;
+    private String recordStatus;
+    private String generalComments;
+    private String entryMethod;
+    private String standIndClass;
+    private String organizationName;
+    private String quickCode;
+    private String facilityId;
+    private String facilityIdAuth;
+    @JsonProperty("street_address_1")
+    private String streetAddress1;
+    @JsonProperty("street_address_2")
+    private String streetAddress2;
+    private String city;
+    private String state;
+    private String stateCode;
+    private String zip;
+    private String county;
+    private String countyCode;
+    private String country;
+    private String addressComments;
+    private String phoneWork;
+    private String phoneExtWork;
+    private String email;
+    private String phoneComments;
+    private String fax;
+    private Long addUserId;
+    private String addUserName;
+    private String addTime;
+    private Long lastChgUserId;
+    private String lastChgUserName;
+    private String lastChgTime;
+    private String refreshDatetime;
+
+    public static OrganizationReporting build(OrganizationSp orgSp) {
```

Comment:
**[BLOCKING]** - Generally it's not recommended to add methods inside model classes. This will make the code not readable and maintainable in the long term. Pull out the methods to separate classes (may be util class).

***THIS IS FOR ALL THE MODEL CLASSES***

Reply:
This uses the builder design pattern. 
https://blogs.oracle.com/javamagazine/post/exploring-joshua-blochs-builder-design-pattern-in-java
The actual post processing is done in separate util classes.

Reply:
But Will move the post processing call to the DataPostProcessor class.

Reply:
Moved all the transform methods into their own transform classes.